networkit/centrality.pyx

```diff
@@ -2928,3 +2932,120 @@ class SciPyPageRank(SpectralCentrality):
 
 		self.eigenvector = spectrum[1][0]
 		self.eigenvalue = spectrum[0][0]
+
+##### from branch ComplexPaths (K. Ahrens)
+
+cdef extern from "<networkit/centrality/ComplexPaths.hpp>" namespace "NetworKit::ComplexPathAlgorithm":
+
+	cpdef enum _Mode  "NetworKit::ComplexPathAlgorithm::Mode":
+		singleNode,
+		allNodes
+
+cdef class ComplexPathMode(object):
+	singleNode = _Mode.singleNode
+	allNodes   = _Mode.allNodes
+
+cdef extern from "<networkit/centrality/ComplexPaths.hpp>":
+	cdef cppclass _ComplexPaths "NetworKit::ComplexPathAlgorithm"(_Algorithm):
+		_ComplexPaths(_Graph G, count threshold, _Mode mode, node start) except +
+		vector[double] getPLci() except +
+		void normalize() except +
+		vector[node] getAdopters() except +
+		_Graph &getComplexGraph() except +
+
+cdef class ComplexPaths(Algorithm):
+	""" 
+	ComplexPaths(G, threshold, mode, start)
+
+	according to:
```

Comment:
Please elaborate more what complex paths does. Take a look at the documentation of other recently implemented algorithms for reference.